shap/plots/_waterfall.py

```diff
@@ -127,7 +127,12 @@ def waterfall(shap_values, max_display=10, show=True):
             neg_lefts.append(loc)
         if num_individual != num_features or i + 4 < num_individual:
             plt.plot(
-                [loc, loc], [rng[i] - 1 - 0.4, rng[i] + 0.4], color="#bbbbbb", linestyle="--", linewidth=0.5, zorder=-1
+                [loc, loc],
+                [rng[i] - 1 - 0.4, rng[i] + 0.4],
+                color=_style.STYLE.vlines_color,
```

Comment:
Right now, this `_style.STYLE.xxx` is a little long and verbose. I'll suggest defining

```python
def plot_xxx():
    style_cfg = get_style()
    # ...
```

at the start of the function definition, then use `style_cfg.primary_color_positive` etc. down below. Seems cleaner to me.

Reply:
Great shout, done